src/nodes/ImageView.vue

```diff
@@ -401,7 +404,7 @@ export default {
 	}
 
 	input[type='text'] {
-		width: 200px;
+		width: 500px;
 		max-width: 80%;
```

Comment:
Sizing doesn't seem to work properly on mobile:

<img width="209" alt="Screenshot 2023-04-26 at 08 34 34" src="https://user-images.githubusercontent.com/3404133/234489459-00592096-2094-4c2f-9017-311ff41959f5.png">

Maybe we can refactor the image__caption__wrapper styling in general a bit, so that it uses a flex layout for the input and delete button where the input spans the full available width.

Reply:
Please check again now.